qrisk/stats.py

```diff
@@ -532,8 +532,9 @@ def information_ratio(returns, factor_returns):
     return np.mean(active_return) / tracking_error
 
 
-def alpha_beta(returns, factor_returns, risk_free=0.0):
-    """Calculates both alpha and beta.
+def alpha_beta(returns, factor_returns, risk_free=0.0, period=DAILY,
+               annualization=None):
```

Comment:
The interaction between period being overwritten by annualization is a bit odd here and probably bears mentioning the doc-string.

@jbredeche Not a big deal here, it's common practice to just use 250 or 252.